templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -339,39 +332,63 @@ Parameters:
     Default: ""
     Description: 'Use custom existing DNS name for your Data Center instance. This will take precedence over HostedZone. Please note: you must own the domain and configure it to point at the load balancer.'
     Type: String
+  DBEngine:
+    Default: 'PostgreSQL'
+    Description: 'Database Engine to use for the application. PostgreSQL or AWS Aurora PostgreSQL'
+    AllowedValues:
+      - 'PostgreSQL'
+      - 'AWS Aurora PostgreSQL'
```

Comment:
Will need to be updated to `Amazon Aurora PostgreSQL` as per AWS preferences in documentation. Also will need to be updated in all the relevant parameter descriptions to be "Amazon Aurora" not "AWS Aurora". This will not be compatible with the latest database metatemplate until then.